finalPrep/annotations.js

```diff
@@ -2,41 +2,41 @@
 // Use the double // to create a new comment
 
 //Build a Bear
-function buildABear(name, age, fur, clothes, specialPower) {
-  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
-  var demographics = [name, age];
-  var powerSaying = "Did you know that I can " + specialPower + " ?";
-  var builtBear = {
-    basicInfo: demographics,
-    clothes: clothes,
-    exterior: fur,
-    cost: 49.99,
-    sayings: [greeting, powerSaying, "Goodnight my friend!"],
-    isCuddly: true,
+function buildABear(name, age, fur, clothes, specialPower) {                    //Creates a function to create a builtBear object with the given peramitors.
+  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;   //Create a greeting interpolating the value passed in the name variable.
+  var demographics = [name, age];                                               //Create the demographics array and assigns the name and age variables to position 0 and 1.
+  var powerSaying = "Did you know that I can " + specialPower + " ?";           //Create the powerSaying variable by concatinating the value passed in through the specialPower variable.
+  var builtBear = {                                                             //Create the builtBear object.
+    basicInfo: demographics,                                                    //Add the values contained in the demographics array to the basic info key of the builtBear object.
+    clothes: clothes,                                                           //Add the value assigned to the clothes variable to the clothes key of the builtBear object.
+    exterior: fur,                                                              //Add the the value assigned to the fur variable to the exterior key of the builtBear object.
+    cost: 49.99,                                                                //Add the static value 49.99 to the cost key of the builtBear object.
+    sayings: [greeting, powerSaying, "Goodnight my friend!"],                   //Add an array containing the values passed through the variables greeting and powerSaying, along with a hard coded string to the sayings key of the builtBear object.
+    isCuddly: true,                                                             //Assign the value true to the isCuddly key of the builtBear object.
   };
 
-  return builtBear
+  return builtBear  //Return the created builtBear object to the funtion that called it.
 }
 
-buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
-buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
+buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares'); //Create a builtBear object with the given values.
```

Comment:
Yes, AND, on the even more foundational level, this line of code `calls the buildABear function`. You'll also see/hear `invokes the buildABear function` being used to describe this line.